README.md

```diff
@@ -1,16 +1,162 @@
 # deroll
 
 Deroll is a TypeScript framework for developing [Cartesi](https://cartesi.io) applications.
-The code below is a minimal application which just loops forever fetching inputs, but with no input handlers. In that case any input is `rejected`.
+The code below is a minimal application which just loops forever fetching inputs, but with no input handlers.
```

Comment:
The "code below" is not exactly below anymore, so it's a bit confusing.

Reply:
fixed